Assets/VRTK/Source/Editor/Attributes/MinMaxRangeDrawer.cs

```diff
@@ -1,4 +1,5 @@
-﻿using UnityEngine;
+﻿#if UNITYEDITOR
```

Comment:
from my understanding, this shouldn't be required since the file resides in an `Editor` folder. Or doesn't Unity recognize this, because the script is in a subfolder of an `Editor` folder?

Reply:
I have the same understand, but in this case it does not work out, as it is supposed to.